package.json

```diff
@@ -36,18 +35,14 @@
     "@types/react": "^17.0.14",
     "@typescript-eslint/eslint-plugin": "^4.28.4",
     "@typescript-eslint/parser": "^4.28.4",
-    "babel-loader": "^8.2.2",
     "eslint": "^7.31.0",
     "eslint-config-google": "^0.14.0",
     "eslint-plugin-jest-dom": "^3.9.0",
     "eslint-plugin-react": "^7.24.0",
     "jest": "^27.0.6",
     "lerna": "^3.22.1",
     "react": "^17.0.2",
-    "react-dom": "^17.0.2",
-    "swc-loader": "^0.1.14",
-    "webpack-bundle-analyzer": "^4.4.2",
-    "webpack-dev-server": "4.0.0-beta.3"
+    "react-dom": "^17.0.2"
```

Comment:
just removed some deps that didn't seem to need to be here (I did this a bit and i'm still learning about all this building, so very might be wrong in many cases...)